javascript2-week2/quizApp2.css

```diff
@@ -0,0 +1,78 @@
+body {
+    font-family: 'Arial', sans-serif;
+    background-color: #f4f4f4;
+    padding: 20px;
+    color: #333;
+}
+
+.quiz-form,
```

Comment:
Now the form is stretched for entire window width wich is not very good for readability. 
General rule is that a for or text block should be of width around 80 characters. There is even a css unit that allows you to define the width of a container in characters. It is called `ch`. Try look it up, add the `max-width` property to these two calees and center the comtainers with `margin: 0 auto`. It will be easier to read. To be able to see the entire queston all the time in on a screen you can change the `input` element for a `textarea` element. See coimment for html file